### PR TITLE
Speed up cursor controller

### PR DESCRIPTION
**Pull request by gfodor**
Speeds up the cursor controller by avoiding A-Frame attribute changes and directly manipulating the underlying three.js data for the line component.

Also fixed up the cursor scale to try to close to a fixed solid angle.

### CODE REVIEW

File: src/components/cursor-controller.js [hunk 1]
```diff
@@ -132,18 +134,38 @@ AFRAME.registerComponent("cursor-controller", {
       camera.object3D.getWorldPosition(cameraPos);
       cameraPos.y = cursor.object3D.position.y;
       cursor.object3D.lookAt(cameraPos);
+      cursor.object3D.scale.setScalar(Math.pow(this.distance, 0.315) * 0.75);
```
Comment (netpro2k) [inline src/components/cursor-controller.js]: How very magic. I am guessing you just eyeballed something that looked ok?

Comment (gfodor) [reply src/components/cursor-controller.js]: yeeeep

**Comment by mquander** [comment]
:tada:

**Comment by gfodor** [comment]
https://github.com/mozilla/hubs/pull/756

**Pull request closed without merging**